DEVELOPMENT.md

```diff
@@ -86,6 +86,28 @@ helmfile -e <service|workload>_cluster -lapp=ingress-nginx -lapp=node-local-dns
 > [!important]
 > To use certificates from Let's Encrypt you must enable [DNS-01 challenges in `cert-manager`](https://cert-manager.io/docs/configuration/acme/dns01/).
 
+Support matrix:
+
+App            | SC | WC | Notes
+-------------- | -- | -- | -----
+calico         | 🟨️ | 🟨️ | Requires Cluster API settings to work.
+cert-manager   | 🟩️ | 🟩️ |
+dex            | 🟩️ | ⬜️ | Does not pull in cert-manager or ingress-nginx.
+falco          | 🟥️ | 🟥️ | Installs but cannot start due to lack of permissions inside Kind.
+fluentd        | 🟩️ | 🟨️ | Installs but cannot start without creating initial resources on WC.
+gatekeeper     | 🟩️ | 🟩️ |
+grafana        | 🟩️ | ⬜️ | Does not pull in cert-manager, dex or ingress-nginx.
+harbor         | 🟩️ | ⬜️ | Does not pull in dex or ingress-nginx.
+hnc            | ⬜️ | 🟩️ |
+ingress-nginx  | 🟩️ | 🟩️ |
+kured          | 🟩️ | 🟩️ |
+node-local-dns | 🟩️ | 🟩️ |
+opensearch     | 🟩️ | ⬜️ | Does not pull ingress-nginx. Prod flavour is heavy on resources.
+prometheus     | 🟩️ | 🟩️ |
+thanos         | 🟩️ | ⬜️ | Does not pull cert-manager, ingress-nginx.
+trivy-operator | 🟩️ | 🟩️ |
+velero         | 🟩️ | 🟩️ |
```

Comment:
Very nice matrix :slightly_smiling_face: 
Nit: you could add definitions on what the colors represent. For me they are self-explanatory, but I could see someone being confused about white in particular. Add that if you want.

Reply:
Questions on support for some minor things in apps:
Does it support alertmanager and calicoaccountant?

Reply:
In the demo you showed me we talked about prometheus rules and that when you just install grafana you don't get all rules. You could add something in the grafana info about which releases to add in order to get all metrics.

Reply:
Calico includes calico-accountant, Prometheus includes alertmanager. so they work. :) Will add notes about dependencies not pulling in resources :+1:

Reply:
Added predefined selectors as a guide.